config/express.js

```diff
@@ -22,7 +22,7 @@ var verifyHandler = function (adminToken, token, tokenSecret, profile, done) {
             redToken: tokenSecret,
             provider: profile.provider,
             name: profile.name,
-            flair: {ptrades: flair1, svex: flair2}
+            flair: {ptrades: flairs[0], svex: flairs[1]}
```

Comment:
Really? That's weird. I'll take a look at it tonight. I remember checking it and it worked great when I tested it.